enterprise_access/apps/content_assignments/content_metadata_api.py

```diff
@@ -60,7 +62,7 @@ def get_human_readable_date(datetime_string, output_pattern=DEFAULT_STRFTIME_PAT
     return None
 
 
-def parse_datetime_string(datetime_string):
+def parse_datetime_string(datetime_string, **set_to_utc):
```

Comment:
You'll want the signature to look like this: `def parse_datetime_string(datetime_string, set_to_utc=False):...`

`**set_to_utc` is doing something else - it's made it so that all optional/extrac kwargs passed to this function end up in a dict stored in the `set_to_utc` argument.
Here's a useful example: https://book.pythontips.com/en/latest/args_and_kwargs.html#usage-of-kwargs